src/Pmi/Security/GoogleGroupsAuthenticator.php

```diff
@@ -163,7 +162,7 @@ public function onAuthenticationSuccess(Request $request, TokenInterface $token,
         // has the user agreed to the system usage agreement this session?
         $request->getSession()->set('isUsageAgreed', false);
         
-        if ($this->app->getConfig('gae_auth')) {
+        if ($this->app->getConfig('local_mock_auth')) {
```

Comment:
This should also check for isLocal. In most places, you check for both the config setting and the local environment, but here and a few other places, you are only checking for the config setting. You might consider adding a helper app method to combine these checks.